superset/__init__.py

```diff
@@ -209,15 +209,17 @@ def index(self):
 results_backend = app.config.get('RESULTS_BACKEND')
 
 # Merge user defined feature flags with default feature flags
-feature_flags = app.config.get('DEFAULT_FEATURE_FLAGS')
-feature_flags.update(app.config.get('FEATURE_FLAGS') or {})
+_feature_flags = app.config.get('DEFAULT_FEATURE_FLAGS')
+_feature_flags.update(app.config.get('FEATURE_FLAGS') or {})
+
+
+def get_feature_flags():
+    return app.config.get('GET_FEATURE_FLAGS')(_feature_flags)
 
 
 def is_feature_enabled(feature):
-    """
-    Utility function for checking whether a feature is turned on
-    """
-    return feature_flags.get(feature)
+    """Utility function for checking whether a feature is turned on"""
+    return get_feature_flags().get(feature)
```

Comment:
As this is coded now, we only allow feature-flags to be booleans. But why can't they be regular variables as well ? Say the default row limit could be a feature flag ?

Reply:
I think as this is coded it allows for more than just booleans. We should have guidelines though.

Reply:
Yeah .. I think it can allow for more than booleans, the guideline can be at the higher layer.